### PR TITLE
CLN: collected cleanups from other branches

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

File: pandas/tests/arithmetic/test_datetime64.py [hunk 1]
```diff
@@ -666,6 +666,7 @@ def test_comparison_tzawareness_compat_scalars(self, op, box_with_array):
     # Raising in __eq__ will fallback to NumPy, which warns, fails,
     # then re-raises the original exception. So we just need to ignore.
     @pytest.mark.filterwarnings("ignore:elementwise comp:DeprecationWarning")
+    @pytest.mark.filterwarnings("ignore:Converting timezone-aware:FutureWarning")
```
Comment (jreback) [inline pandas/tests/arithmetic/test_datetime64.py]: is ignoring this warning actually hiding something?

File: pandas/tests/groupby/test_categorical.py [hunk 2]
```diff
@@ -508,7 +508,7 @@ def test_datetime():
     desc_result = grouped.describe()
 
     idx = cats.codes.argsort()
-    ord_labels = cats.take_nd(idx)
+    ord_labels = cats.take(idx)  # TODO: deprecate take<->take_nd alias
```
Comment (jreback) [inline pandas/tests/groupby/test_categorical.py]: can you make an issue (and remove the comment)

**Review by jreback — CHANGES_REQUESTED** [review]
comments

File: pandas/tests/groupby/test_categorical.py (hunk 2 above)
Comment (jbrockmendel) [reply pandas/tests/groupby/test_categorical.py]: done

File: pandas/tests/arithmetic/test_datetime64.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/arithmetic/test_datetime64.py]: AFAICT it is just noise in the logs

**Comment by jreback** [comment]
I think this is fine, but can you rebase anyways

**Review by jreback — APPROVED** [review]

**Comment by jbrockmendel** [comment]
Rebased and green

**Pull request merged**